### PR TITLE
PR for BSIP-63 (Payment Channels)

### PR DESCRIPTION
**Pull request by christophersanborn**
Addresses issue #172

### CODE REVIEW

File: bsip-0063.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+# Discussion
+
+## Risks to channel operators
+
+### Chain halts or other system-wide extended outages
+
+In the event of a system-wide chain halt, there is a risk that channel state-update and refund transactions will expire before the chain resumes, essentially erasing channel state.  In this case, funds will be locked in escrow in the payment-channel account.  Because Alice and Bob are both signers on that account, they can, in principal, negotiate a settlement transaction to disburse the funds as intended, although they do run the risk of deadlock if they cannot come to an agreeable course of action.  Wallet software can be coded to respond to this situation automatically and fairly with a channel extension procedure, so that users can expect a fair outcome so long as both parties are using un-modified wallet software.
```
Comment (pmconrad) [inline bsip-0063.md]: Typo: princip~~al~~*le*

File: bsip-0063.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
...r efficiently handling high volumes of incremental transactions between two parties with a minimum of on-chain transactions, thus reducing blockchain load and reducing fees paid by the two parties.  The methodology covers the mechanics of opening and funding a channel (on-chain), exchanging state-update transactions (off-chain), optionally rebalancing or extending the lifetime of a channel (on-chain), and finally closing a channel (on-chain).  This document does _NOT_ specify details of the communication layer or the messaging protocal between parties.  When combined with a suitable communication layer and messaging protocol, this payment channel methodology can be developed into a full payment channel protocol, which can be used to bring features such as streaming micropayments and connectivity to the Interledger network to the BitShares platform, which opens up possibilities for cross-ledger payments and swaps, and may allow for the development of new [trustless or nearly-trustless gateways](https://whaleshares.io/@christopher/bitshares-and-interledger-ilp-summit-summary-for-the-bitshares-community) for deposits and withdrawels of off-chain assets for their on-chain counterparts.
```
Comment (pmconrad) [inline bsip-0063.md]: Typo: withdraw~~e~~*a*l

File: bsip-0063.md (hunk 1 above)
Comment (christophersanborn) [reply bsip-0063.md]: Corrected.

File: bsip-0063.md (hunk 2 above)
Comment (christophersanborn) [reply bsip-0063.md]: Corrected, as well as a few others.

**Pull request merged**